10up/simple-page-ordering
# Add: Pre-commit hooks

### Description of the Change
Add pre-commit hooks and linters to keep the coding style of this project consistent and prevent potential syntax error early.


### Alternate Designs
n/a


### Benefits
New code is checked before committing.


### Possible Drawbacks
We may need to update Github action workflows to match with new changes.


### Verification Process
- See new files `composer.json` and `composer.lock` in repo.
- Make test changes to `simple-page-ordering.php`, stage it, commit it, see the pre-commit hooks run.


### Checklist:






### Applicable Issues
Closes #40 


### Changelog Entry
